docs(readme): Add breaking changes notes (#117)
stackbuilders/react-native-spotlight-tour: README.md

This library handles animations at the native level and is perfect
for the following:

- Guiding users on how to use your application
- Showing an introduction to your users

<span><img src="docs/rnst-bounce.gif" alt="spotlight-bounce-gif" width="300"/></span>
<span><img src="docs/rnst-fade.gif" alt="spotlight-fade-gif" width="300"/></span>
<span><img src="docs/rnst-slide.gif" alt="spotlight-slide-gif" width="300"/></span>

## Requirements

- [ReactJS](https://reactjs.org/) >= 16.8.0
- [React Native](https://reactnative.dev/) >= 0.50.0
- [react-native-svg](https://github.com/react-native-svg/react-native-svg) >= 12.1.0

## Install

With `npm`:

```bash
$ npm install react-native-spotlight-tour
```

With `yarn`:

```bash
$ yarn add react-native-spotlight-tour
```

## 🚨 Breaking changes: v2 to v3

This major update brings a few fixes, some great new features, and some breaking changes. These are some highlight you'll need to consider while upgraging from v2 to v3:

> - The package has been rename from `@stackbuilders/react-native-spotlight-tour` to just `react-native-spotlight-tour`

Reviewer: ```suggestion
- The package has been renamed from `@stackbuilders/react-native-spotlight-tour` to just `react-native-spotlight-tour`
```

```

With `yarn`:

```bash
$ yarn add react-native-spotlight-tour
```

## 🚨 Breaking changes: v2 to v3

This major update brings a few fixes, some great new features, and some breaking changes. These are some highlight you'll need to consider while upgraging from v2 to v3:

- The package has been renamed from `@stackbuilders/react-native-spotlight-tour` to just `react-native-spotlight-tour`
  - Dont worry, this library is still developed and maintained by the [Stack Builders Inc.](https://www.stackbuilders.com/) team!
  - Remove the former package from your dependencies and use the command described in the [Install section](#install)
  - Rename any import from the previous name to use just `react-native-spotlight-tour` instead
- Tooltip positioning was refactored
  - Props related to the tooltip position were removed from `SpotlightTourProvider` and the `TourStep` object.
    - Both `Align` and `Position` enums were removed
    - Both `alignTo` and `position` props were removed
  - We now delegate the positioning to [FloatingUI](https://floating-ui.com/), so you can use the `floatingProps` prop to configure its global behavior or granularly on each step.

>   - Middleware functions are re-xported from `@floating-ui/react-native` so you can directly import them from `react-native-spotlight-tour`.

Reviewer 2: ```suggestion
  - Middleware functions are re-exported from `@floating-ui/react-native` to `react-native-spotlight-tour`.
```

Author: Nice catch! Thank you @well1791!
